ceph/nvmeof/initiators/linux/__init__.py

```diff
@@ -8,6 +8,8 @@ def __init__(self, node):
         super().__init__(node)
         self.node = node
         self.configure()
+        if self.distro_version() == "10.0":  # BZ#2415011
```

Comment:
Nice comment message, it does give perspective on why the change is implemented. It would be great if the `commit` message is also updated.

Reply:
Given that the issue may arise on any platform, I believe we should check before calling `self.configure_hostnqn()` 

Reference: [BZ2049991](https://bugzilla.redhat.com/show_bug.cgi?id=2049991)